### DISCUSSION TITLE
overhead?

### DISCUSSION THREAD

**Discussion by glushakov** [weaponry/pgscv#14] [General]
Hi.
1.Didn't you count the overheads when using pgscv with default settings?
2. Which  is default frequency update? On graphics it looks like about 15 seconds. Interval cannot be changes?

**Comment by lesovsky** [comment]
Hi,

> 1.Didn't you count the overheads when using pgscv with default settings?

No, didn't. Overhead depends on number of databases/tables/indexes/statements within the instance. If there are too many objects and scrape time become too long, it is worth to review the config and maybe disable collecting metrics from secondary databases.

> 2. Which is default frequency update? On graphics it looks like about 15 seconds. Interval cannot be changes?

pgSCV polls stats only when configured to send metrics to Weaponry. In basic scenarios it is scraped by Prometheus accordingly to its [scrape_config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) (see scrape_interval).

**Reply by glushakov** [reply]
ok, thanks!
